### PR TITLE
Experience the marshmallow

### PR DESCRIPTION
**Pull request by samatabulusu**
Please review this, @casey-collab

### CODE REVIEW

File: english/marshmallow.md [hunk 1]
```diff
@@ -12,8 +12,12 @@ When you look away, a marshmallow hand attempts to grab the lighter.
 
 The realization suddenly dawns upon you the marshmallow is self aware!
 
+Hold the image of the marshmallow and..
+
 Do you:
 
+[Close your eyes](experience/mediate.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/experience/experience.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+you close your eyes and gaze at the point between the eyebrows
```
Comment (casey-collab) [inline english/experience/experience.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by samatabulusu** [comment]
fixed broken link..please review again @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**